Formula/a/apache-flink-cdc.rb

```diff
@@ -0,0 +1,82 @@
+class ApacheFlinkCdc < Formula
+  desc "Flink CDC is a streaming data integration tool"
+  homepage "https://nightlies.apache.org/flink/flink-cdc-docs-stable"
+  url "https://www.apache.org/dyn/mirrors/mirrors.cgi?action=download&filename=flink/flink-cdc-3.1.0/flink-cdc-3.1.0-bin.tar.gz"
+  mirror "https://archive.apache.org/dist/flink/flink-cdc-3.1.0/flink-cdc-3.1.0-bin.tar.gz"
+  sha256 "f10b26a2368fd933c26b928fc118f1f1e28efec3ab58b0ec0c58f3f22906a9ab"
+  license "Apache-2.0"
+  head "https://github.com/apache/flink-cdc.git", branch: "master"
+
+  depends_on "apache-flink" => :test
+
+  resource "mysql-connector" do
+    url "https://search.maven.org/remotecontent?filepath=org/apache/flink/flink-cdc-pipeline-connector-mysql/3.1.0/flink-cdc-pipeline-connector-mysql-3.1.0.jar"
+    sha256 "2503a60a7e97c48a8b739d5cdb7270a93ba0d48bf073f9b88e1b34c915c4dd88"
+  end
+  resource "paimon-connector" do
+    url "https://search.maven.org/remotecontent?filepath=org/apache/flink/flink-cdc-pipeline-connector-paimon/3.1.0/flink-cdc-pipeline-connector-paimon-3.1.0.jar"
+    sha256 "799c723beb31379cb093a8caf01e402a126a8b6e5674d5cd579e819221e8fae8"
+  end
+  resource "kafka-connector" do
+    url "https://search.maven.org/remotecontent?filepath=org/apache/flink/flink-cdc-pipeline-connector-kafka/3.1.0/flink-cdc-pipeline-connector-kafka-3.1.0.jar"
+    sha256 "285de33acad088ec3b6da51bd58ab9e6326626ea025b48d9c048d2383ca2df6e"
+  end
+  resource "doris-connector" do
+    url "https://search.maven.org/remotecontent?filepath=org/apache/flink/flink-cdc-pipeline-connector-doris/3.1.0/flink-cdc-pipeline-connector-doris-3.1.0.jar"
+    sha256 "20d7744376ac152994e7de803100a792a1c583a5ca03a647c55e9039ae59237c"
+  end
+  resource "starrocks-connector" do
+    url "https://search.maven.org/remotecontent?filepath=org/apache/flink/flink-cdc-pipeline-connector-starrocks/3.1.0/flink-cdc-pipeline-connector-starrocks-3.1.0.jar"
+    sha256 "48ce5ffe3462c965f39ed3a97717f96739e3160759a5c6422b86ed151d6165ba"
+  end
+  resource "values-connector" do
+    url "https://search.maven.org/remotecontent?filepath=org/apache/flink/flink-cdc-pipeline-connector-values/3.1.0/flink-cdc-pipeline-connector-values-3.1.0.jar"
+    sha256 "7e16714c1395fa3139e60f189de54877020369e4ec2a7ec13cc4c7b84a26402e"
+  end
+
+  def install
+    # Install launch script
+    libexec.install "bin"
+    bin.write_exec_script libexec/"bin/flink-cdc.sh"
```

Comment:
```suggestion
    bin.write_exec_script libexec/"bin/flink-cdc.sh" => "flink-cdc"
```

Reply:
Seems `write_exec_script` doesn't support creating alias. Added a `mv` to trim `.sh` extension.

Reply:
I see 👍